.github/workflows/set_rook_ceph_cluster.yaml

```diff
@@ -0,0 +1,52 @@
+name: setup-ceph-cluster
+on:
+  pull_request:
+    branches:
+      - main
+      - release-*
+
+defaults:
+  run:
+    # reference: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell
+    shell: bash --noprofile --norc -eo pipefail -x {0}
+
+# cancel the in-progress workflow when PR is refreshed.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  rook-ceph-cluster:
```

Comment:
can we change this to ceph-cluster?

Reply:
can you also update the file name to have `setup_ceph_cluster`?

Reply:
done